src/client/components/my-factoids.js

```diff
@@ -0,0 +1,32 @@
+const h = require('react-hyperscript');
+const { Link } = require('react-router-dom');
+const { Component } = require('react');
+
+
+class MyFactoids extends Component {
+  render(){
+    let factoids = this.props.factoids.map(factoid => {
+      return h('div.factoid-entry', [
+        h(Link, {
+          className: 'plain-link',
+          to: `/document/${factoid.id}/${factoid.secret}`
```

Comment:
If the secret is null, would this go to `/document/some-id/null`?

Reply:
it goes to ```document/id/read-only```

Reply:
It might be better to use `document/id` without any secret appended to the end at all